### PR TITLE
[PR] Updating README and BUILD.

### LINKED ISSUE #190: Run the `MVP` on `localhost`
Our objective is to make the **`MVP`** as ultra-beginner-friendly to Devs 
_especially_ those who are **`new`** `Elixir/Phoenix` ...
That means if there are _any_ assumptions about prior knowledge/experience 
we either need to clarify them in _this_ repo or link to where they can learn/understand more
in our _many_ other projects. 🤞 

Following on from #140 @LuchoTurtle please:

+ [x] clone the repo to your `localhost` 💻 
+ [x] #191
  + [x] Get the `AUTH_API_KEY` from https://auth.dwyl.com 🔑 

+ [x] If there are gaps in the `README.md` please:
  + [x] note them in this issue. 💬 
  + [x] create a PR to update them. 🙏 

again, our goal is for _anyone_ to be **up-and-running** with the **`MVP`** in **_less_ than 5 minutes** including `mix setup` and `AUTH_API_KEY` ... so the completeness/quality of the docs is mega important!

# `.then` ...

+ [x] Read through the `BUILDIT.md` file and just sense check to see if we missed anything.
  Again the objective is to give people a _complete_ guide for re-creating the **`MVP`** from scratch.

If you have **_any_ questions** please ask! ❓ 

For clarity, the purpose of this exercise is to familiarise yourself with the **`MVP`** as much as possible.
I will ask you to build a feature for it today. That issue/task will follow shortly. ⏳

### PR DESCRIPTION
**Pull request by LuchoTurtle**
fixes #190 

As requested, here's the PR of the changes to README and possibly BUILD while running the MVP.

### CODE REVIEW

**Comment by LuchoTurtle** [comment]
Just got through replicating the project and I fixed/clarified some aspects as I was reading, implementing, and testing.

Overall, it's a great read and very detailed, so not much was needed to change. But sometimes I was stumped because I didn't know where I should use some code snippets. It should be more clear now 😄

File: BUILDIT.md [hunk 1]
```diff
@@ -168,14 +169,12 @@ That tells us everything is working as expected. 🚀
...
-file to exclude `Phoenix` files from `excoveralls` checking.
-Add alias (shortcuts) in `mix.exs` `defp aliases do` list. 
-
-e.g: `mix c` runs `mix coveralls.html` 
-see: [**`commits/d6ab5ef`**](https://github.com/dwyl/app-mvp/pull/90/commits/d6ab5ef7c2be5dcad7d060e782393ae29c94a526) ...
+file. You need to do two things: 
+*add a [`coveralls.json`](https://github.com/dwyl/app-mvp/blob/main/coveralls.json) 
+file (copy the contents)* 
+and _change `mix.exs` file according to 
+[**`this commit`**](https://github.com/dwyl/app-mvp/pull/90/commits/d6ab5ef7c2be5dcad7d060e782393ae29c94a526)
```
Comment (nelsonic) [inline BUILDIT.md:176]:
```suggestion
[**`commit #d6ab5ef7c`**](https://github.com/dwyl/app-mvp/pull/90/commits/d6ab5ef7c2be5dcad7d060e782393ae29c94a526)
```

Comment (LuchoTurtle) [reply BUILDIT.md:176]: Force of habit, sorry!

File: BUILDIT.md [hunk 2]
```diff
@@ -295,7 +294,7 @@ end
 Now if you refresh the page 
 you should see the following:
 
-![liveveiw-page-with-tailwind-style](https://user-images.githubusercontent.com/194400/176137805-34467c88-add2-487f-9593-931d0314df62.png)
```
Comment (nelsonic) [inline BUILDIT.md:298]: Markdown syntax for images is fine as the M⬇️  parser will render it to HTML. 👌 

Also, consider constraining the viewport to the minimum size to reduce bandwidth and scrolling.
e.g:
liveview

Could be reduced by 60% to:
liveview

Sill gives people the exact same info but doesn't consume their bandwidth/time.

**Review by nelsonic — APPROVED** [review]
@LuchoTurtle Looks great! Thanks! :shipit:

File: BUILDIT.md (hunk 2 above)
Comment (nelsonic) [reply BUILDIT.md:298]: E.g: 
liveview-cropped

**Pull request merged**